main.cpp

```diff
@@ -779,6 +780,9 @@ int main(int argc, char** argv)
     
     printf("done in %f4.2 sec\n", ((float)(end-start))/1000000000.0);
     printf("create tx loop in %6.2f sec\n", ((float)(end-createTxLoopStart))/1000000000.0);
+    printf("Generate a block and push <Enter>... ");
```

Comment:
I think we want to system() a script (configured in txunami.json) here.  That script can just wait for an enter, or it can wait while polling for a block.

Reply:
Yeah a simple script like the one in [point 7](https://read.cash/@jtoomim/when-and-how-to-spam-scalenet-90643e9b) by Mr Toomim would probably suffice. 

I have encountered instances where only part of my fan-out transactions would confirm in the first block, even in two stage setups. 

For now I'm just monitoring getmempoolinfo to know when to push Enter.